maturity-model.md

```diff
@@ -0,0 +1,175 @@
+* PP-3: There is an standard process for collective product and plans definition.
+
+
+**Development Process & Tools**
+
+* DP-0:	Each team follows its own development process and tools. They are not defined to share knowledge and artifacts outside development team. Siloed development teams.
+* DP-1: Development teams use shared code repositories, internally. Some teams develop their own CI process, using non corporate or standard CI tools. There is no code review process defined, although some projects teams do it internally.
+* DP-2: The organization sponsors and promotes a shared repository for collective knowledge. Some teams develop their own CI process, using corporate CI tools. There are CI environments. Code review process defined, and used by some projects. Code review is sometimes done by external team members.
+* DP-3: Most teams develop their own CI process, using corporate CI tools. There are CI environments. Code review process defined, and used by some projects. Code review is usually done by external team members.
+
+**Decisions**
+
+* DC-0: Decision-makers often withhold data and resources without explanation.
+* DC-1: Materials that are part of decision-making practices become available for review after decisions are finalized.
+* DC-2:	People feel like they know about—and are helping to shape—most (but not all) important decisions as those decisions are unfolding. Materials that are part of decision-making practices are available at defined project milestones
+* DC-3: People feel like they are a part of a shared, standard process for collective decision-making that the organization endorses. Materials that are part of decision-making practices are continuously accessible during work processes.
+
+**Helpful Resources**
+
+* RS-0: Individuals and teams neither contribute to nor draw upon a shared repository of knowledge.
+* RS-1:	Individuals and teams release project materials for review internally, after they've finished their work. Individuals and teams share resources, but in disconnected, fragmented, or individualized/siloed systems or repositories. Individuals and teams share resources, but there is no commonly expressed or shared understanding of the criteria used to determine whether information is sensitive or not. Do not "share thinking on others".
+* RS-2: Individuals and teams make project-related materials accessible to some members of project teams according to clearly defined and shared formats and/or protocols. Individuals and teams withhold sensitive data and resources, provide limited details, context, and scope.
+* RS-3: Individuals and teams make project-related materials broadly accessible to the organization—and possibly outside the organization as well—according to clearly defined and shared formats and/or protocols. Individuals and teams who must withold sensitive data and resources are clear about what they're not sharing, and others understand why those materials are not available to them.
+
+**Stories**
```

Comment:
While this is important for InnerSource to thrive I wonder if this could be phrased more specifically for InnerSource or at least be extended by an explanation of why that is important? Thinking about it, the "why is this point important" likely should be added to all of these points.

Reply:
Perhaps experiences?

The rationale for this point is that it is important to learn from each others, no matters if these are failures or successes.

Reply:
I tried to re-phrase and add as a rationale. Feedback welcome.